src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/LoopDetector.kt

```diff
@@ -204,7 +204,7 @@ internal class LoopDetector(
         // Has the thread changed? Reset the counters in this case.
         check(lastExecutedThread == iThread) { "reset expected!" }
         // Ignore coroutine suspension code locations.
-        if (codeLocation == COROUTINE_SUSPENSION_CODE_LOCATION) return Decision.Idle
```

Comment:
Why UNKNOWN? The documentation here and of the `UNKNOWN_CODE_LOCATION` field says it is the coroutine suspension code location.

Reply:
It was `COROUTINE_SUSPENSION_CODE_LOCATION` previously, I renamed it into `UNKNOWN_CODE_LOCATION` because I re-used it for in a different context, but with a similar purpose (that is to represent an unknown code location). 

I fixed the comments in the code so they now mention `UNKNOWN_CODE_LOCATION`.

Reply:
It doesn't. The code now looks like this:
```
        // Ignore unknown code locations.
        if (codeLocation == UNKNOWN_CODE_LOCATION) return Decision.Idle
```